contributing_to_docs/term_glossary.adoc

```diff
@@ -34,109 +37,128 @@ An action consists of _project_, _verb_, and _resource_:
 * *Project* is the project containing the resource that is to be acted upon.
 * *Verb* is a get, list, create, or update operation.
 * *Resource* is the API endpoint being accessed. This is distinct from the
-referenced resource itself, which can be a pod, deployment config, build, etc.
+referenced resource itself, which can be a Pod, DeploymentConfig, Build, etc.
```

Comment:
I didn't think that Build was a Kubernetes entity. Are we capitalizing it anyway?

Reply:
I called out both Kubernetes and OpenShift API objects in the note about it, so by that logic it would fit, yeah.

Reply:
DeploymentConfig is an OpenShift object too, FWIW.